cellxgene_schema_cli/tests/fixtures/examples_validate.py

```diff
@@ -142,7 +142,7 @@
             "HANCESTRO:0575",
             "HsapDv:0000003",
             "donor_1",
-            "nucleus",
+            "na",
```

Comment:
diff is from https://github.com/chanzuckerberg/single-cell-curation/issues/827 merged into main